scripts/one-offs/2020_04_28_backfill_intial_rounds_for_all_games.sql

```diff
@@ -0,0 +1,8 @@
+DO $$
+DECLARE 
+    t_row games%rowtype;
+BEGIN
+    FOR t_row in (SELECT DISTINCT games.id FROM games LEFT JOIN rounds ON games.id = rounds.game_id WHERE rounds.game_id IS NULL) LOOP
+        INSERT INTO rounds (game_id, value, order_sequence) VALUES (t_row.id, 'taboo', 0), (t_row.id, 'charades', 1), (t_row.id, 'password', 2);
+    END LOOP;
+END $$;
```

Comment:
There should be 3x rounds as games.

![image](https://user-images.githubusercontent.com/2534903/80535292-681a4900-8955-11ea-8611-738177ed03d2.png)

![image](https://user-images.githubusercontent.com/2534903/80535318-71a3b100-8955-11ea-9fd4-beed2ef7a278.png)